src/api.js

```diff
@@ -1,30 +1,47 @@
-const API_BASE_URL = process.env.REACT_APP_SERVER_URL;
+const API_BASE_URL = "https://api.zujonow.com";
+const VIDEOSDK_TOKEN = process.env.REACT_APP_VIDEOSDK_TOKEN;
+const API_AUTH_URL = process.env.REACT_APP_AUTH_URL;
 
 export const getToken = async () => {
-  const res = await fetch(`${API_BASE_URL}/get-token`, {
-    method: "GET",
-  });
-
-  const { token } = await res.json();
-  return token;
+  if(VIDEOSDK_TOKEN !=="" && API_AUTH_URL !== ""){
```

Comment:
No need to check empty value.
You could also chek this way

`if (VIDEOSDK_TOKEN && API_AUTH_URL)
{
console.log('Both constant contain some value')
}
`